test/battleship/game/scoreboard_test.go

```diff
@@ -25,17 +26,17 @@ func TestGetScoreBoardWithOneCellLongShip(t *testing.T) {
 	expected := &scoreboard.ScoreBoard{3, cells}
 
 	// When computing its possible positions
-	// on a 3x3 grid
-	actual, _ := grid.GetScoreBoard(3, ship) // adresse récupérée
+	actual, _ := game.GetScoreBoard(grid, ship) // adresse récupérée
```

Comment:
GetScoreBoard is not tested in the right file. GetScoreBoard is in grid.go and tests are in scoreboard_test.go

Reply:
Moved GetScoreBoard to the right file so the tests match